com.unity.robotics.urdf-importer/Runtime/Extensions/UrdfGeometryCollision.cs

```diff
@@ -307,25 +319,43 @@ private static void ConvertMeshToColliders(GameObject gameObject, string locatio
                     List<Mesh> colliderMeshes = decomposer.GenerateConvexMeshes(meshFilter.sharedMesh);
                     foreach (Mesh collider in colliderMeshes)
                     {
+                        var c = collider;
```

Comment:
Is it strictly necessary to declare a new variable here?

Reply:
Foreach iteration variable 'collider' is immutable.
So yes, apparently.
I'm going to go ahead and swap 'c' and 'collider' around however.